includes/rest-api/Controllers/Version2/class-wc-rest-orders-v2-controller.php

```diff
@@ -166,10 +166,12 @@ protected function get_order_item_data( $item ) {
 			}
 		}
 
-		// Add SKU and PRICE to products.
+		// Add SKU, PRICE, IMAGE and VIRTUAL flag to products.
```

Comment:
Those changes can't be made in v2 since this endpoint is obsolete, it need to be moved into v3, where we could still include some features like that.

Reply:
Thank you for the review @claudiosanches! I'm not sure how to go about migrating this to v3 😅 It looks data for v3 endpoint is still fetched from v2 controller, which is why I added it there